spec/integration/recipes/recipe_index_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'Recipe', type: :system do
```

Comment:
👍 Good work on incorporating Capybara for integration testing! 👍 To enhance your test coverage, consider expanding your test cases beyond checking the current path. Verify the content and functionality on the page to ensure a more comprehensive evaluation of your application. Here are some suggestions:

- [ ] **Check Recipe Names:** Confirm that the page displays the names of recipes belonging to the current user.

- [ ] **Verify Link Functionality:** Test the functionality of links on the page. Ensure that links, buttons, or elements related to recipes or navigation are working correctly.

- [ ] **Validate Content:** If there's additional content on the page, such as descriptions, images, or details about each recipe, validate their presence and correctness.

Here's an example of how you might expand one of your test cases:

```ruby
context 'user logs in successfully' do
  it 'should redirect to the root path and display user recipes' do
    login_user(@user)
    
    expect(current_path).to eq(root_path)
    expect(page).to have_content('Signed in successfully.')

    # Additional checks for content
    expect(page).to have_content('My Recipes') # Assuming there's a section heading
    expect(page).to have_link('Recipe 1') # Replace with an actual recipe name
    expect(page).to have_link('Recipe 2') # Replace with another recipe name
    # Add more checks as needed
  end
end
```

By including checks for content and functionality, you ensure that your integration tests cover a broader range of scenarios, making your testing more robust. Keep up the good work! 🚀🌐